contracts/State.sol

```diff
@@ -11,6 +12,10 @@ interface IVerifier {
     ) external view returns (bool r);
 }
 
+interface ISMT {
```

Comment:
Let's rename `SMT` -> `Smt`, `ISMT` -> `ISmt`. Easier to ream IMHO

Reply:
renamed